### PR TITLE
docs: programmatic deployment blogpost

### PR DESCRIPTION
**Pull request by afmika**
Blogpost to help discover programmatic deployment additions.

#### Migration notes

...

- [ ] The change comes with new or modified tests
- [ ] Hard-to-understand functions have explanatory comments
- [ ] End-user documentation is updated to reflect the change

## Summary by CodeRabbit

- **New Features**
  - Introduced programmatic deployment feature for deploying typegraphs within the Metatype ecosystem using TypeScript/Python SDKs.
  - Added new configuration options and deployment functions to enhance automation and flexibility in deployment processes.

- **Documentation**
  - Added a new blog post detailing the programmatic deployment feature and its usage.

### CODE REVIEW

**Comment by zifeo** [comment]
Can you add how to upgrade to the 0.4.x series as a last step in the article?

File: website/blog/2024-05-09-programmatic-deployment/index.mdx [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+  baseUrl: "<TYPEGATE_URL>",
+  auth: new BasicAuth("<USERNAME>", "<PASSWORD>"),
+  secrets: { POSTGRES: "<DB_URL>" },
+  artifactsConfig,
+};
+```
+    </TabItem>
+</SDKTabs>
+
+### Deploy/Remove
```
Comment (zifeo) [inline website/blog/2024-05-09-programmatic-deployment/index.mdx:77]: lowercase r?

File: website/blog/2024-05-09-programmatic-deployment/index.mdx [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+
+Just like any other dependency in your favorite programming language, each SDKs can be installed with your favorite package manager.
+
+You can use one of the commands below to get started with the latest available version.
+
+
+<SDKTabs>
+    <TabItem value="python">
+```bash
+pip install typegraph
```
Comment (zifeo) [inline website/blog/2024-05-09-programmatic-deployment/index.mdx:34]: @destifo `--upgrade` do not forget to that

**Review by Natoandro — DISMISSED** [review]

**Review by Yohe-Am — DISMISSED** [review]

**Review by zifeo — APPROVED** [review]
👏

**Pull request merged**